ovos_utils/gui.py

```diff
@@ -765,42 +781,48 @@ def _pages2uri(self, page_names: List[str]) -> List[str]:
         LOG.debug(f"Resolved pages: {page_urls}")
         return page_urls
 
+    @staticmethod
+    def _normalize_page_name(page_name: str) -> str:
+        """
+        Normalize a requested GUI resource
+        @param page_name:
+        @return:
+        """
+        if isfile(page_name):
+            LOG.warning("GUI resources should specify a resource name and not "
+                        "a file path. This may cause unexpected behavior.")
+            return page_name
+        file, ext = splitext(page_name)
+        if ext == ".qml":
+            log_deprecation("GUI resources should exclude gui-specific file "
+                            "extensions. In the future, this may cause "
+                            "unexpected behavior.", "0.1.0")
```

Comment:
i think devs wont understand this, should provide a explicit example of self.gui.show_page

Reply:
There are a couple methods that could log this (show/remove page/s), but the deprecation log should include the line where that call is made. Updated the log to show what string to replace